subdaos/overview.md

```diff
@@ -14,12 +14,15 @@ In most respects, subDAOs are self-standing, unique DAOs. Their semi-independent
 
 ## SubDAO Types
 
-SubDAOs are divided into two types:
+SubDAOs are divided into three types:
+
+- **[FacilitatorDAOs](facilitator.md)** are tasked with organizing the internal mechanisms of MakerDAO, allocatorDAOs, and miniDAOs.
+- **[AllocatorDAOs](allocator.md)** can generate Dai directly from MakerDAO and allocate it as they see fit across the broader DeFi ecosystem. They also manage public-facing entry points to the Maker Ecosystem and are able to spin off miniDAOs.
+- **[MiniDAOs](mini.md)** are spun off by allocatorDAOs as outgrowths of an idea or product they are interested in further decentralizing, promoting, or otherwise consolidating. MiniDAOs are experimental offshoots and may be very short-lived.
+
+A further, higher-level division is drawn between miniDAOs and both facilitatorDAOs and allocatorDAOs, whereby the latter are Major SubDAOs (they are incubated directly by MakerDAO) and the former are Minor SubDAOs (they are incubated by allocatorDAOs) &mdash; i.e., facilitatorDAOs and allocatorDAOs are Major SubDAOs; miniDAOs are Minor SubDAOs.
```

Comment:
```suggestion
FacilitatorDAOs and AllocatorDAOs are incubated directly by MakerDAO and are called Major SubDAOs while MiniDAOs are incubated by AllocatorDAOs and are called Minor SubDAOs.
```

Reply:
For style purposes, let's capitalize the first word of the SubDAO? e.g FacilitatorDAO and not facilitatorDAO